tests/test_utils.py

```diff
@@ -49,7 +49,13 @@ def test_results_function(
         ) -> None:
             pass
 
-        results_api = ResultsAPI(app_id=test_constants.VALID_RESULTS_API["app_id"])
+        with patch(
+            "veracode.api.get_app_id",
+            return_value=test_constants.VALID_UPLOAD_API["app_id"],
```

Comment:
It looks like we're mixing `VALID_UPLOAD_API` and `VALID_RESULTS_API` here and below, please make sure they're aligned
```suggestion
            return_value=test_constants.VALID_RESULTS_API["app_id"],
```

Reply:
Should be fixed now.